backend/Squirrel.ConsoleApp/Extensions/ExceptionFilterExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Net;
+using Squirrel.ConsoleApp.Models;
+using Squirrel.ConsoleApp.Models.Models;
+
+namespace Squirrel.ConsoleApp.Extensions;
+
+public static class ExceptionFilterExtensions
+{
+    public static (HttpStatusCode statusCode, ErrorCode errorCode) ParseException(this Exception exception)
+    {
+        return exception switch
+        {
+            FileNotFoundException _ => (HttpStatusCode.NotFound, ErrorCode.FileNotFound),
+            InvalidOperationException _ => (HttpStatusCode.NotFound, ErrorCode.FileDamage),
+            _ => (HttpStatusCode.InternalServerError, ErrorCode.General),
+        };
+    }
```

Comment:
we already have such Extension in Common project

Reply:
Same, we don't need references from another solution

Reply:
ok